README.md

```diff
@@ -3,226 +3,169 @@
-    })();
-    ```
-
-### Run ###
+- Install `prebid.js` as an `npm` dependency of your project
+- Duplicate `node_modules/prebid.js/adapters.json` to under your project path, e.g. `path/to/your/list-of-adapters.json`
+- Remove the unnecessary adapters
+- Run the `prebid.js` build under the `node_modules/prebid.js/` folder
+
+        $ gulp build --adapters path/to/your/list-of-adapters.json
+
+Most likely your custom `prebid.js` will only change when there's:
+
+- A change in your list of adapters
+- A new release of `prebid.js`
+
+Having said that, you are probably safe to check your custom bundle into your project.  You can also generate it in your build process.
+
+<a name="Run"></a>
+
+## Run
+
+To configure Prebid.js to run locally, edit the example file `./integrationExamples/gpt/pbjs_example_gpt.html`:
+
+1. Change `{id}` values appropriately to set up ad units and bidders
+2. Set the path to Prebid.js in your example file as shown below (see `pbs.src`):
+    + For development:
+
+            (function() {
+                    var d = document, pbs = d.createElement('script'), pro = d.location.protocol;
+                    pbs.type = 'text/javascript';
+                    pbs.src = ((pro === 'https:') ? 'https' : 'http') + './build/dev/prebid.js';
+                    var target = document.getElementsByTagName('head')[0];
+                    target.insertBefore(pbs, target.firstChild);
+            })();
+ 
+    + For deployment:
+
+            (function() {
+                    var d = document, pbs = d.createElement('script'), pro = d.location.protocol;
+                    pbs.type = 'text/javascript';
+                    pbs.src = ((pro === 'https:') ? 'https' : 'http') + './build/dist/prebid.js';
+                    var target = document.getElementsByTagName('head')[0];
+                    target.insertBefore(pbs, target.firstChild);
+            })();
+
+To run the project locally, use:
 
     $ gulp serve
 
-This runs code quality checks, generates prebid files and starts a webserver at
-`http://localhost:9999` serving from project root. Navigate to your example implementation to test,
-and if your prebid.js file is sourced from the `./build/dev` directory you will have sourcemaps
-available in browser developer tools.
+This runs code quality checks, generates all the necessary files and starts a web server at `http://localhost:9999` serving from the project root. Navigate to your example implementation to test, and if your `prebid.js` file is sourced from the `./build/dev` directory you will have sourcemaps available in your browser's developer tools.
+
+To run the example file, go to:
+
++ `http://localhost:9999/integrationExamples/gpt/pbjs_example_gpt.html` 
+
+To view a test coverage report, go to:
+
++ `http://localhost:9999/build/coverage/karma_html/report`
+
+A watch is also in place that will run continuous tests in the terminal as you edit code and tests.
+
+<a name="Contribute"></a>
+
+## Contribute
```

Comment:
Can you add a section here on unit tests / PRs? Just call out that all PRs require a minimum of 80% unit test code coverage.